app.js

```diff
@@ -0,0 +1,34 @@
+
+const { request, response } = require('express')
+const express = require('express')
+
+//Init Express
+const app = express()
+
+//2. End points/routes handlers
+
+//make everything inside of public/available
+app.use(express.static('public'))
+
+//our first Route:
+app.get('/home', (request, response, next) => {
+    response.sendFile(__dirname + '/views/home.html')
+})
+
+app.get('/about', (request, response, next) => {
+    response.sendFile(__dirname + '/views/about.html')
+})
+
+app.get('/work', (request, response, next) => {
+    response.sendFile(__dirname + '/views/work.html')
+})
+
+
+
+
+
+//Listeter on a port
+
+app.listen(3001, () => {
```

Comment:
Aquí estaría bien especificar en que puerto se esta corriendo nuestra app c:
```suggestion
app.listen(3001, () => {
    console.log("server online in port 3001")
})
```